Day-4-5/src/App.js

```diff
@@ -1,14 +1,17 @@
-import React, { createElement } from "react";
+import React, { useState } from "react";
 import ReactDOM from "react-dom/client";
 import Cards from "./components/Cards";
 import Header from "./components/Header";
-import { TeamList } from "./data/data";
+import SearchBar from "./components/SearchBar";
+import TeamList from "./data/data.json";
 
 const App = () => {
+  const [filterRestaurants, setFilterRestaurants] = useState(TeamList);
   return (
     <>
       <Header members={TeamList.length} />
-      <Cards data={TeamList} />
+      <SearchBar searchRestaurants={setFilterRestaurants} />
```

Comment:
Don't change the name of setters to such a generic name. It creates confusion in big code bases.